liquibase-standard/src/main/java/liquibase/ui/UIServiceEnum.java

```diff
@@ -0,0 +1,20 @@
+package liquibase.ui;
+
+import lombok.Getter;
+
+/**
+ * Enum used to define the available UIServices provided by Liquibase that can be set using global parameters
+ */
+@Getter
```

Comment:
If you really want to lombokify you can could also do `@AllArgsConstructor` or maybe just `@Constructor`

Reply:
Nice, this one wasn't suggested by Sonar